Makefile

```diff
@@ -610,6 +610,8 @@ edpm_deploy_baremetal_prep: export EDPM_TOTAL_NODES=${DATAPLANE_TOTAL_NODES}
 edpm_deploy_baremetal_prep: export EDPM_NETWORK_CONFIG_TEMPLATE=${DATAPLANE_NETWORK_CONFIG_TEMPLATE}
 edpm_deploy_baremetal_prep: export EDPM_SSHD_ALLOWED_RANGES=${DATAPLANE_SSHD_ALLOWED_RANGES}
 edpm_deploy_baremetal_prep: export EDPM_CHRONY_NTP_SERVER=${DATAPLANE_CHRONY_NTP_SERVER}
+edpm_deploy_baremetal_prep: export EDPM_REGISTRY_URL=${DATAPLANE_REGISTRY_URL}
+edpm_deploy_baremetal_prep: export EDPM_CONTAINER_TAG=${DATAPLANE_CONTAINER_TAG}
```

Comment:
`make edpm_deploy_baremetal` doesn't work right now without these lines added

Reply:
I am proposing this same fix in another PR to try to get it merged sooner to unblock the baremetal Zuul job: https://github.com/openstack-k8s-operators/install_yamls/pull/468.  So if this PR merges first, I'll just close that one.

Reply:
Hmm...`edpm_deploy_baremetal` target is not used in the zuul job. That's only for deploying with existing BMHs.

Reply:
Ah, okay.  So something else might be causing repeated failures there.